src/store/place/index.ts

```diff
@@ -119,7 +119,7 @@ for(let i=0; i<10; i++)
 {
     generated_places.push(
         {
-            id: generate_id(),
+            id: prev_generate_id(),
```

Comment:
unnessary renaming

Reply:
same. prev_generated_id is for random restaurant generation